src/components/gamespec/BoardTile.tsx

```diff
@@ -1,10 +1,17 @@
-import React from 'react';
+import * as React from 'react';
 import { blue500 } from 'material-ui/styles/colors';
 import { GridTile } from 'material-ui/GridList';
 
 import styles from '../../styles';
 
-const BoardTile = props => {
+interface BoardTileProps {
+  keyProp: any;
+  selectedKey: any;
+  handleGridTileClick: (keyProp: any, image: any) => void;
+  image: any;
+}
```

Comment:
Using any doesn't provide any benefits of TypeScript. If possible, can you move away from any?